### ISSUE TITLE
Problem of Replacement of "BatchNorm" to "GroupNorm"

### ISSUE THREAD

**Issue by CinKKKyo** [Res2Net/Res2Net-PretrainedModels#74]
Hi, Dr. Gao, I have a question about model modification. Due to my work setting, I could only set batch_size=1, so I am going to modify the BatchNorm to GroupNorm. Based on the original code, I just changed BatchNorm to GroupNorm directly and set the group=8, actually, it doesn't work, do you have any advice for this replacement?
